.github/workflows/ci.yml

```diff
@@ -23,6 +25,13 @@ jobs:
     - name: Build E2E Runner
       shell: cmd
       run: dotnet.exe build src\Ztm.EndToEndTests
+    - name: Install main database
+      shell: cmd
+      run: choco.exe install postgresql --version=10.6.0.20190131 --params '/Password:postgres' --params-global
+    - name: Migrate main database
```

Comment:
Same here. My idea is `Create Main Database` looks better due to we did not migrate anything.